src/lib/tabs/tab-label-wrapper.ts

```diff
@@ -19,7 +19,8 @@ export const _MdTabLabelWrapperMixinBase = mixinDisabled(MdTabLabelWrapperBase);
  * @docs-private
  */
 @Directive({
-  selector: '[md-tab-label-wrapper], [mat-tab-label-wrapper]',
+  selector: '[md-tab-label-wrapper], [mat-tab-label-wrapper], ' +
```

Comment:
Isn't this directive purely internal? If so, we should be able to just remove the dash-case selectors.